src/Findologic/Config/FinSearchConfigEntity.php

```diff
@@ -13,35 +13,18 @@ class FinSearchConfigEntity extends Entity
 {
     use EntityIdTrait;
 
-    /**
-     * @var string
-     */
-    protected $configurationKey;
-
-    /**
-     * @var mixed
-     */
+    protected string $configurationKey;
+
+    /** @var mixed */
     protected $configurationValue;
```

Comment:
Couldn't you declare types for this variable?

Reply:
It can basically be any type, that's why I kept using `mixed`

Reply:
`mixed` is not available as variable type and union types are still not supported (`array|int|boolstring`)